content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -1,75 +1,80 @@
-   them to fill in the fields. Now, if you want to change it instantly, it is possible to do so
-   directly from the quotation or the sales order.
+On the :guilabel:`Create Contact` pop-up form, start by clicking the default :guilabel:`Other
+Address` field to reveal a drop-down menu of address-related options.
 
-Add addresses from a contact form
-=================================
+Select any of the following options:
 
-Previously, we talked about the contact form that you can fill in directly from a quotation or a
-sales order to add billing and shipping addresses to customers. But, if you go to
-:menuselection:`Sales --> Orders --> Customers`, you can create or modify every customer you want
-and add, delete or modify invoice and delivery addresses instantly there, before creating a
-quotation.
+- :guilabel:`Contact`
+- :guilabel:`Invoice Address`
+- :guilabel:`Delivery Address`
+- :guilabel:`Other Address`
+- :guilabel:`Private Address`
 
-.. image:: different_addresses/addresses_3.png
+Once an option is selected, proceed to enter the corresponding contact information that should be
+used for the specified address type.
+
+.. image:: different_addresses/create-contact-window.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to add addresses from a contact form on Odoo Sales?
+   :alt: Create a new contact/address on a contact form.
+
+Then, click :guilabel:`Save & Close` to save the address and close the :guilabel:`Create Contact`
+window. Or, click :guilabel:`Save & New` to save the address and immediately input another one.
 
-Deal with different addresses
-=============================
+Address added to quotations
+===========================
 
-Like for the previous example, go to :menuselection:`Sales --> Orders --> Customers` and create a
-new customer. There, you can add company information but, more importantly, you can enter billing
-and shipping addresses under the **Contacts & Addresses** tab.
+When a customer is added to a quotation, the :guilabel:`Invoice Address` and :guilabel:`Delivery
+Address` fields autopopulate with the corresponding addresses specified on the customer's contact
+form.
 
-.. image:: different_addresses/addresses_4.png
+.. image:: different_addresses/quotation-address-autopopulate.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to manage different customer addresses on Odoo Sales?
+   :alt: Invoice and Delivery Addresses autopopulate on a quotation.
 
-Once done, you can return to your Sales dashboard and create a new quotation. Now, if you enter
-your new customer, you will see that the other fields will fill in by themself with the information
-previously saved for billing and shipping addresses.
+The :guilabel:`Invoice Address` and :guilabel:`Delivery Address` can also be edited directly from
+the quotation by clicking the :guilabel:`Edit` button, and then clicking the :guilabel:`External
+link` buttons next to each address line (represented by an arrow in a square).
```

Comment:
This button is an internal link, not an external, and I think since the text only shows on the hover tool-tip it would be more appropriate to use the icon
```suggestion
the quotation by clicking the :guilabel:`Edit` button, and then clicking the :guilabel:`➡️ (right
arrow)` internal link buttons next to each address line.
```